Automattic/wp-calypso
# A/B/n Experiment: Add FAQ section in the plans step of signup flow

#### Proposed Changes

* End the previous plans step experiment `calypso_signup_plans_step_optimize_202208_v1`. 
* Add an FAQ section in the plans step and retain the feature presentation UI from https://github.com/Automattic/wp-calypso/pull/67151.
* Verify that the copy for the Q & A matches the one in the sheet pdgrnI-1CY#comment-2873.

This is setup as an A/B/n experiment, so there are two treatment UIs being tested:

**Treatment 1**


![calypso localhost_3000_start_plans_intervalType=yearly](https://user-images.githubusercontent.com/1269602/189869314-c3064cf5-fce5-4d87-b37f-6724beda9c8c.png)

**Treatment 2**
![calypso localhost_3000_start_plans (1)](https://user-images.githubusercontent.com/1269602/189947776-7211144b-4bbf-47b1-a964-888d744605bd.png)



#### Testing Instructions



* Assign yourself to `treatment_condensed` and `treatment_expanded` in the experiment `calypso_signup_plans_step_faq_202209_v1` in Abacus.
* Signup for a new account and verify that the plans step UI matches the one shown in screenshots above for each of the treatments.
* Verify that a Tracks event is fired when you open/close an FAQ question with the appropriate ID.
* Verify that a Tracks event is fired when the "learn more" link is clicked on the answer for "Can you build a site for me?"
* Verify that mobile view is unchanged - this experiment is only for desktop, so mobile view should have no UI changes.
* Verify that non-EN is unaffected
* Verify that control version in unaffacted.
* Verify that you can continue to complete signup and create a site / complete a purchase.
* Verify that Calypso `/plans` page is unaffected, and the FAQ section in that page is also unaffected.
